--- gupnp-1.2.6/examples/get-volume.c.orig	2021-05-24 16:34:52.798355300 +0200
+++ gupnp-1.2.6/examples/get-volume.c	2021-05-30 09:12:04.462878396 +0200
@@ -50,7 +50,7 @@
                 &error);
 
         if (error != NULL) {
-                g_critical (error->message);
+                g_critical ("%s", error->message);
                 g_clear_error (&error);
         }
 
@@ -111,7 +111,7 @@
         g_list_free (out_names);
 
         if (error != NULL) {
-                g_critical (error->message);
+                g_critical ("%s", error->message);
                 g_clear_error (&error);
         } else {
                 g_print ("Current volume: %s\n",
@@ -147,7 +147,7 @@
         GUPnPContext *context = gupnp_context_new ("wlp3s0", 0, &error);
 
         if (error != NULL) {
-                g_error (error->message);
+                g_error ("%s", error->message);
         }
 
         GUPnPControlPoint *cp = gupnp_control_point_new (context, CONTENT_DIR);
